### ISSUE TITLE
The package name is incorrect in AUR info

### ISSUE THREAD

**Issue by f2404** [archlinuxfr/yaourt#216]
```
$ yaourt -Si afx2usb-linux-git
Repository     : aur
Name           : -git
Version        : 0.r4.9f2ef7d-1
```

**Comment by larchunix** [comment]
Yaourt doesn't support well PKGBUILD when a custom variable is used in a field.

```bash
_pkgname=afx2usb-linux
pkgname=$_pkgname-git
```

This can be fixed by using package-query as in #215.

**Comment by larchunix** [comment]
Similar issue for the version of `dnf` package:

`pkgver` is defined like this:

```bash
_pkgver=1.1.6
_rpmrel=1
pkgver=$_pkgver.$_rpmrel
```

As a result:

    yaourt -Si dnf
    Repository     : aur
    Name           : dnf
    Version        : .-1

**Comment by f2404** [comment]
Oh I see. Looks like I'll have to deal with the array-like fields in my PR.

**Comment by larchunix** [comment]
Fixed by #215.
